force_wfmanager/ui/setup/system_state.py

```diff
@@ -0,0 +1,31 @@
+from traits.api import (
+    HasTraits, Instance, Unicode, Callable
+)
+from force_wfmanager.ui.setup.new_entity_creator import NewEntityCreator
+
+
+class SystemState(HasTraits):
```

Comment:
New class acts as a temporary reference to currently selected objects. Not strictly necessary, but abides by the principle of least knowledge guidelines for cross-communicating between tasks / panes